pymeasure/adapters/visa.py

```diff
@@ -121,18 +121,20 @@ def ask_values(self, command):
         """
         return self.connection.query_values(command)
 
-    def binary_values(self, command, header_bytes=0, dtype=np.float32):
+    def binary_values(self, command, header_bytes=None, dtype=np.float32):
```

Comment:
I would suggest breaking this change out into a separate PR. It will make it much easier to get merged.

Reply:
#259